devices/tuya/_TZ3000_ja5osu5g_smart_button.json

```diff
@@ -46,15 +46,21 @@
           "name": "attr/uniqueid"
         },
         {
-            "name": "config/battery",
-            "parse": {
-                "ep": 1, "cl": "0x0001", "at": "0x0021",
-                "eval": "Item.val = Attr.val / 2"
-            },
-            "read": {
-                "ep": 1, "cl": "0x0001", "at": "0x0021"
-            },
-            "refresh.interval": 7300
+          "name": "config/battery",
+          "awake": true,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "eval": "Item.val = Attr.val / 2"
+          },
+          "default": 0
```

Comment:
You can remove the default value, as this is taken from the generic. More importantly, please re-add the refresh interval. Rule of thumb: if there's a read function defined, `refresh.interval` must be present (otherwise the device gets hammered with read requests).

Lastly, you may also add the function to the parse part for consistency.

Reply:
>if there's a read function defined, `refresh.interval `must be present.

Ha yes, right, realy important.
3 correctives dones.